features/pom.xml

```diff
@@ -70,7 +70,6 @@
                 <configuration>
                     <descriptors>
                         <descriptor>file:${project.build.directory}/feature/camel-features.xml</descriptor>
-                        <descriptor>file:${project.build.directory}/feature/spring-features.xml</descriptor>
```

Comment:
This is true thanks to Karaf 4.4.6. It's ok for ne but we have to clearly state that the minimum Karaf required version is 4.4.6 (whereas before, camel-karaf 4.x works with any karaf 4.x version).
Maybe worth to keep the spring features in camel for now ?

Reply:
OK, fair enough. Let's do that.